docs/tutorials/subscription-platform.md

```diff
@@ -401,11 +401,22 @@ In `fxa-auth-server/config/secrets.json`, set the following config values under
     }
 }
 ```
-Use the same values for `keyFilename` and `projectId` as used for `authFirestore` config in [Testing IAP subscriptions locally](#testing-iap-subscriptions-locally).
 
-:::note
-To make API calls to the [Google Play Developer API](https://developer.android.com/google/play/developer-api), the service account referenced by the keyfile [must have the API enabled](https://developers.google.com/android-publisher/authorization). This is not required for most Google IAP development, so consider skipping that step.
-:::
+You can stop here if you're only going to use the mock IAP script for your local development and do not need a real connection to the Play API.
```

Comment:
Can we add a permalink to the script file or a link to where this mock IAP script is explained elsewhere in these docs?

Reply:
Sure thing